docs/user/exercises/quiz.rst

```diff
@@ -1,2 +1,440 @@
+
+- Resolve a Quiz Exercise: Students can solve quiz exercises.
+
+Prepare a Quiz Exercise
+--------
+This section describes the process to:
+
+    - Create Quiz Exercises
+    - Create Drag and Drop Model Quiz Exercises
+    - Import Quiz Exercises from other courses
+    - Export Quiz Exercises
+
+Create new Quiz Exercise
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- On the Artemis home page, click on the Course Management button |CourseManagementButton|.
+
+- Navigate into **Exercises** of a specific course by clicking on the exercise button.
+
+    .. figure:: quiz/course-dashboard.png
+            :align: center
+
+- In the quiz exercises section, click on the Create Quiz button |CreateAQuizButton| to open the following form to create the new quiz exercise.
+
+    .. figure:: quiz/create-a-quiz-form.png
+            :align: center
+
+    - **Title**: Provide a title for the quiz exercise (The red line means that this field is mandatory).
+
+    - **Categories**: Type the category for the quiz exercise.
+
+    - **Difficulty**: Select the difficulty level among Easy, Medium and Hard. It is possible to select No level.
+
+    - **Duration**: Provide the time in minutes and seconds for students to solve the quiz exercise.
+
+    - **Options**: Choose between presenting the questions in Random order or not.
+
+    - **Batch Mode**: Batch Mode controls how students can participate in the quiz.
+
+        - Synchronized: There is exactly one opportunity for all students to participate in the quiz.
+
+        - Batched: There are multiple opportunities for students to participate in the quiz by using a password.
+
+        - Individual: Students can participate in the quiz by themselves at any time while it is released.
+
+    - **Visible from**: The date and hour when the quiz becomes visible to students.
+
+    - **Schedule Quiz Start**: To establish the date and hour at which the quiz will be available to solve be solved.
+
+        - **Start of working time**: Set the time when the students are able to see the questions and start answering. Students can start working on the quiz from this time until the duration ends.
+
+    - **Should this exercise be included in the course score calculation?**:
+
+        - Yes: the points will be included in the course score calculation.
+
+        - No: the points will not be included in the course score calculation.
+
+        - Bonus: the points will be considered as bonus points.
+
+    - **Questions**: There are four ways to add questions to a quiz exercise.
+
+        - **Add Multiple-Choice Question**
+
+            This kind of question is composed of a problem statement with multiple options.
+
+            .. figure:: quiz/multiple-choice-question.png
+                :align: center
+
+            - Short question title: Provide a short title (Mandatory).
+
+            - Points: Assign the value points for this question.
+
+            - Scoring type:
+
+                - All or nothing
```

Comment:
```suggestion
                - All or Nothing
```

Reply:
The option in Artemis is established as "All or Nothing"